packages/client/src/modules/play/context/playContext.tsx

```diff
@@ -220,6 +254,7 @@ function useGameSocket({
     setGameWithTeams,
     setPlayerActions,
     setActionPointsLimitExceeded,
+    setPlayer,
```

Comment:
React Hook useEffect has a missing dependency: 'player'. Either include it or remove the dependency array

Reply:
J'ai ajouté `players` pour tester mais le problème est qu'à chacune de ses updates, ça crée une nouvelle socket sans fermer la précédente. `players` va être mise à jour fréquemment, j'ai préféré l'enlever du tableau de dépendances plutôt que de fermer la socket pour en ouvrir une nouvelle à chaque fois

Reply:
@Baboo7 ok. Rajoute un TODO et je regarderai comment faire quand tout sera merger.